docs/SplunkOperatorUpgrade.md

```diff
@@ -23,13 +23,19 @@ A Splunk Operator for Kubernetes upgrade might include support for a later versi
 ```
 wget -O splunk-operator-install.yaml https://github.com/splunk/splunk-operator/releases/download/1.1.0/splunk-operator-install.yaml
 ```
+
+2. Download the upgrade script.
```

Comment:
Exchange steps 2 & 3 just to keep consistency i.e when we say review the file it may be easier for the user to follow that they need to review `splunk-operator-install.yml`